dms-service/README.md

```diff
@@ -85,25 +85,26 @@ swim:
 
 ### Pattern
 
-The fields `filename-coo-pattern`, `incoming-name-pattern`, `filename-overwrite-pattern` and `verify-procedure-name-pattern` require a specific syntax (inspired by the sed command and regex substitution).
+The `*-pattern`-fields require a specific syntax (inspired by the sed command and regex substitution).
 
 ```
 s/<regex>/<substitution>/<options>
 ```
 
 The pattern is applied as following:
-- `<regex>` is applied to input
+- `<regex>` is applied to input (filename without extension)
+  - The extension is re-added where required
```

Comment:
Can we be more specific here? Let's add in brackets "(the extension is needed when creating the content object in order for the dms to determine the correct file type") or similar